spring-auto-restdocs-core/pom.xml

```diff
@@ -89,7 +94,6 @@
             <groupId>org.apache.tomcat.embed</groupId>
             <artifactId>tomcat-embed-el</artifactId>
             <version>9.0.2</version>
-            <scope>test</scope>
```

Comment:
Is there a reason why this was changed? I think that the test scope is sufficient. We want to force as few dependencies on Spring Auto REST Docs clients as possible.

Reply:
Absolutely. Can't remember why I have removed the `test` scope. I think I have tried something... I have added it scope again now.